frontend/src/bundles/common/components/confirm-hire/confirm-hire.tsx

```diff
@@ -0,0 +1,114 @@
+import { useAppForm, useCallback, useState } from '../../hooks/hooks.js';
+import {
+    Button,
+    ConfirmModal,
+    FormControl,
+    Grid,
+    RadioGroup,
+    Typography,
+} from '../components.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+    label: string;
+    modalLabel: string;
+    onSubmit: (payload: { isHired: boolean }) => void;
+};
+
+const options = [
+    {
+        value: 'true',
+        label: 'Yes',
+    },
+    {
+        value: 'false',
+        label: 'No',
+    },
+];
+
+const ConfirmHire: React.FC<Properties> = ({ label, modalLabel, onSubmit }) => {
+    const { control } = useAppForm<{ check: string }>({
+        defaultValues: { check: 'false' },
+    });
+
+    const [isHired, setIsHired] = useState(false);
+    const [isSubmitStep, setIsSubmitStep] = useState(false);
+
+    const handleToSubmitStep = useCallback(() => {
+        setIsSubmitStep(true);
+    }, []);
+
+    const handleModalClose = useCallback(() => {
+        setIsSubmitStep(false);
+    }, []);
+
+    const handleConfirm = useCallback(() => {
+        onSubmit({ isHired });
+        handleModalClose();
+    }, [isHired, onSubmit, handleModalClose]);
+
+    const handleFormSubmit = useCallback(
+        (event_: React.BaseSyntheticEvent): void => {
+            event_.preventDefault();
+            const check = event_.target.check.value;
+
+            switch (check) {
+                case 'true': {
+                    setIsHired(true);
+                    break;
+                }
+                case 'false': {
+                    setIsHired(false);
+                    break;
+                }
+                default: {
+                    return;
+                }
+            }
+
+            handleToSubmitStep();
+        },
+        [handleToSubmitStep],
+    );
```

Comment:
I am not sure that we need this logic

Reply:
Maybe it can be realized in another way, but I don't know how.
It works as follow
submit form -> get checked value (STRING) -> convert ( BOOLEAN ) -> save in _state_ -> open modal -> modal operate with _state_ boolean value

Reply:
get proper value, set this value and submit

Reply:
```js
const options = [
    {
        value: 'yes',
        label: 'Yes',
    },
    {
        value: 'no',
        label: 'No',
    },
];

const isHired = check === "yes" ? true : false
```